src/poll.js

```diff
@@ -434,19 +438,33 @@ class Poll {
     let scheduleText = this.formatScheduleStatus()
     let isInactive = true
     let msg = this.renderBase(isInactive).responseType('ephemeral')
-    let actionsAttachment = msg
+    let actionsAttachmentDaily = msg
```

Comment:
I would make this a bit more flexible like we discussed. You can replace this block with:

```
    let actionAttachment = null

    module.exports.periods.forEach((period, i) => {
      if (i % 4 === 0) {
        actionAttachment = msg.attachment()
          .fallback('Repeat Frequency')
          .callbackId('in_or_out_callback')
          .mrkdwnIn(['text'])
       }
      actionAttachment.action()
        .name(`repeat_${period}`)
        .value(value)
        .text(`${radio(period)} ${self.formatPeriod(period)}`)
        .type('button')
    })
```